Tests/XcodeGenKitTests/SchemeGeneratorTests.swift

```diff
@@ -193,32 +193,35 @@ class SchemeGeneratorTests: XCTestCase {
             }
 
             $0.it("generates target schemes from config variant") {
-                let configVariants = ["Test", "Production"]
+                let configVariants = ["Test", "PreProd", "Prod"]
                 var target = app
                 target.scheme = TargetScheme(configVariants: configVariants)
                 let configs: [Config] = [
                     Config(name: "Test Debug", type: .debug),
-                    Config(name: "Production Debug", type: .debug),
+                    Config(name: "PreProd Debug", type: .debug),
+                    Config(name: "Prod Debug", type: .debug),
                     Config(name: "Test Release", type: .release),
-                    Config(name: "Production Release", type: .release),
+                    Config(name: "PreProd Release", type: .release),
```

Comment:
I think this will fail the tests, and I'm sure it's not uncommon to have that lowercase
```suggestion
                    Config(name: "PreProd release", type: .release),
```

Reply:
@yonaskolb took me a while to get the point, then it totally made sense!

Should be fixed now as you originally suggested: whitespace tokenization and lowercase compare